modules/aws/eks/eks.tf

```diff
@@ -51,6 +54,63 @@ resource "aws_eks_cluster" "this" {
     }
   }
 
+  tags       = local.global_tags
+  depends_on = [aws_cloudwatch_log_group.this]
+}
+
+
+
+# CloudWatch Encrytion
+resource "aws_kms_key" "cloudwatch_encryption" {
```

Comment:
Should encryption be enabled by default?

Reply:
I don't see why not. My understanding is that it doesn't come with any cost.